neuro-cli/src/neuro_cli/config.py

```diff
@@ -48,7 +53,10 @@ async def show(root: Root) -> None:
             jobs_capacity = {}
     quota = await root.client.users.get_quota()
     org_quota = await root.client.users.get_org_quota()
-    root.print(fmt(root.client.config, jobs_capacity, quota, org_quota))
+    config_cluster = None
+    if energy:
+        config_cluster = await root.client._clusters.get_cluster(cluster_name)
```

Comment:
where do we read the cluster information from?

I see that we also render price_kwh in that output table. `/api/v1/config` does not output neither price nor co2.

Reply:
This is taken from https://github.com/neuro-inc/neuro-config-client, see trace ([1](https://github.com/neuro-inc/neuro-cli/blob/ys/support-energy-schedule/neuro-sdk/src/neuro_sdk/_client.py#L30), [2](https://github.com/neuro-inc/neuro-cli/blob/ys/support-energy-schedule/neuro-sdk/src/neuro_sdk/_clusters.py#L126))

As far as I could see, we were transitioning between deprecating and removing neuro-config-client-related functionality within SDK and using dedicated neuro-config-client for this.